Assets/Mirror/Runtime/NetworkReaderExtensions.cs

```diff
@@ -236,7 +236,21 @@ public static List<T> ReadList<T>(this NetworkReader reader)
             }
             return result;
         }
-
+        
+        public static HashSet<T> ReadHashSet(this NetworkReader reader)
+        {
+            int length = reader.ReadInt();
+            if(length < 0)
+                return null;
+            HashSet<T> result = new HashSet<T>(length);
+            for(int i = 0 ; i < length ; i++)
+            {
+                result.Add(reader.Read<T>());
+            }
+            return result
```

Comment:
Missing a semicolon here, you'll get a compile error otherwise.

Reply:
Fixed